src/main/java/Duke.java

```diff
@@ -19,41 +20,23 @@ public void markAsDone() {
     }
     public static void main(String[] args) {
         Scanner input = new Scanner(System.in);
-        int count = 0;
+        int count = 0; //total number of tasks in the list
         Task[] Tasks = new Task[100];
-
-        System.out.println("____________________________________________________________");
-        System.out.println(" Hello! I'm Duke");
-        System.out.println(" What can I do for you?");
-        System.out.println("____________________________________________________________");
+        Greet();
```

Comment:
Hi, just thought that since this is a method, it should start with small letters instead of capital letters, small issues